awsmp/models.py

```diff
@@ -139,3 +142,139 @@ def ami_access_role_arn_validator(cls, value):
         if not value.startswith("arn:aws:iam::"):
             raise ValueError(f"{value} is invalid role format. Please check your role again.")
         return value
+
+
+class DescriptionModel(BaseModel):
+    ProductTitle: str
+    ShortDescription: str
+    LongDescription: str
+    Sku: str
+    Highlights: List[str]
+    SearchKeywords: List[str]
+    Categories: List[str]
+
+
+class PromotionalResourcesModel(BaseModel):
+    LogoUrl: HttpUrl
+    Videos: List[HttpUrl]
+    AdditionalResources: List[
+        dict[
+            str,
+            str,
```

Comment:
shouldn't that be  a https://docs.pydantic.dev/2.2/api/networks/#pydantic.networks.HttpUrl field?